server/repository/src/db_diesel/report_row.rs

```diff
@@ -46,6 +46,9 @@ table! {
       comment -> Nullable<Text>,
       sub_context -> Nullable<Text>,
       argument_schema_id -> Nullable<Text>,
+      is_custom -> Bool,
+      version -> Text,
+      code -> Nullable<Text>,
```

Comment:
```suggestion
      code -> Text,
```
Lets go with not null?